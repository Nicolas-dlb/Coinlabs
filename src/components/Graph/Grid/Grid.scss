.grid {
  display: flex;
  pointer-events: none;
  height: 100%;
  width: 100px;
  position: relative;
  right: 30px;
  bottom: -6px;
  .grid_card {
    opacity: 0.07;
    border-left: 1px solid lightgrey;
    border-right: 1px solid lightgrey;
    height: 9vh;
    width: 18px;
    position: relative;
    bottom: 10vh;
    left: 14px;
    pointer-events: none;
  }
}

@media only screen and (max-height: 680px) {
  .grid {
    bottom: 3px !important;
  }
}
@media only screen and (max-height: 580px) {
  .grid {
    bottom: 8.5px !important;
  }
}

.grid-container {
  position: relative;
  bottom: 44vh;
  width: 100%;
  pointer-events: none;
}
.grid2 {
  position: relative;
  width: 110%;
  height: 2vh;
  left: 3vw;

  display: flex;
  flex-direction: column;
  justify-content: center;
  margin-top: 3vh;
  pointer-events: none;

  line {
    opacity: 0;
    margin-top: 20px;
    pointer-events: none;
  }
  .line2 {
    position: relative;
    top: 2vh;
    pointer-events: none;
  }
}
