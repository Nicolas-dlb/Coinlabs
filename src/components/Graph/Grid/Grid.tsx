import "./Grid.scss";
import React from "react";

function Grid() {
  return (
    <div className="grid">
      <span className="grid_card" />
      <span className="grid_card" />
      <span className="grid_card" />
      <span className="grid_card" />
      <span className="grid_card" />
    </div>
  );
}

export default Grid;
