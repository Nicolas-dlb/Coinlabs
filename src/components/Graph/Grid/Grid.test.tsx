import React from "react";
import { shallow } from "enzyme";
import Grid from "./Grid";

describe("Grid", () => {
  it("should render Grid component", () => {
    const GridComponent = shallow(<Grid />);
    expect(GridComponent).toMatchSnapshot();
  });
  it("should render 5 element with grid_card classname", () => {
    const GridComponent = shallow(<Grid />);
    expect(GridComponent.find(".grid_card")).toHaveLength(5);
  });
});
