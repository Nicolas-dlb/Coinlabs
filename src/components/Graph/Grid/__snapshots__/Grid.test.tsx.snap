// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Grid should render Grid component 1`] = `
<div
  className="grid"
>
  <span
    className="grid_card"
  />
  <span
    className="grid_card"
  />
  <span
    className="grid_card"
  />
  <span
    className="grid_card"
  />
  <span
    className="grid_card"
  />
</div>
`;
